xds/build.gradle

```diff
@@ -39,11 +38,18 @@ dependencies {
 
     testCompile project(':grpc-core').sourceSets.test.output
 
-    compileOnly libraries.javax_annotation
+    compileOnly libraries.javax_annotation,
+            // At runtime use the epoll included in grpc-netty-shaded
```

Comment:
Technically it would probably be best to do the same `minus()` thing here, but that can be a follow-up.